tests/LINEBot/EventRequestParserTest.php

```diff
@@ -587,7 +596,11 @@ public function testParseEventRequest()
             $this->assertEquals('replytoken', $event->getReplyToken());
             $this->assertEquals('contentid', $event->getMessageId());
             $this->assertEquals('text', $event->getMessageType());
-            $this->assertEquals('message (love)', $event->getText());
+            $this->assertEquals('@example Hello, world! (love)', $event->getText());
```

Comment:
And please not to reuse existing assertions.

Reply:
Sorry, fixed.